inc/reminder.class.php

```diff
@@ -103,7 +103,7 @@ function canUpdateItem() {
     * @since 0.85
     *
     * @see CommonDBTM::canPurgeItem()
-   **/
+    **/
```

Comment:
Such changes makes PRs difficult to run, please do not apply editor changes on a whole file ;)